templates/etc/systemd/system/kubelet.service.j2

```diff
@@ -35,6 +35,7 @@ ExecStart={{k8s_bin_dir}}/kubelet \
   --cni-conf-dir={{k8s_cni_conf_dir}} \
   --cni-bin-dir={{k8s_cni_bin_dir}} \
   --serialize-image-pulls={{k8s_worker_kubelet_serialize_image_pulls}} \
+  --fail-swap-on=false \
```

Comment:
Not sure about this option. There is a lengthy discussion here https://github.com/kubernetes/kubernetes/issues/7294  about swap and why to disable it by default. Can you please make this a variable and add it `defaults/main.yml` defaulting to `false`? This way everyone could override the setting to his needs.

Reply:
Ok, no problem